doc/samples/SqlConnection_OpenAsync_ContinueWith.cs

```diff
@@ -0,0 +1,43 @@
+using System;
```

Comment:
I would leave out any samples that use the old Begin*/End* async pattern. We don't want to encourage any new use of those in MDS.

Reply:
This sample relates to [this](https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/asynchronous-programming#adding-the-new-asynchronous-feature-in-an-existing-application-mixing-old-and-new-patterns) section that I'm porting to ADO.Net documents.
Do you want me to remove that section too or just removing the samples is enough?